bindata/etcd/pod.yaml

```diff
@@ -390,3 +392,7 @@ ${COMPUTED_ENV_VARS}
     - hostPath:
         path: /etc/kubernetes
       name: config-dir
+    - hostPath:
+        path: /var/lib/etcd-auto-backup
+        type: DirectoryOrCreate
```

Comment:
that's a cool feature, what permissions is it creating the directory with?

Reply:
wdy mean ?!!

Reply:
`/var/lib/etcd-auto-backup` doesn't exist, so it has to be created: with what permissions?
it's kinda important that we don't have 0777 permissions on etcd backups

Reply:
so the docs says it creates it with `0755`. 

But I can just remove this and create the dir with the correct permission as [here](https://github.com/openshift/cluster-etcd-operator/blob/518174a5a82c923029c7c375c9b6d06d03c83f30/bindata/etcd/pod.yaml#L26) 

wdyt ?

Reply:
I think 0755 is too broad, 0640 seems good enough for backups. They shouldn't be publicly readable.

Reply:
fixed https://github.com/openshift/cluster-etcd-operator/pull/1306/commits/fe29a0c9208151c7cf49d6017c6eb9bc8bea7278